CIScripts/BuildStage.ps1

```diff
@@ -9,6 +9,10 @@ $Credentials = New-Object PSCredentialT ($Username, $Password)
 $NothingToBuild = $Env:COMPONENTS_TO_BUILD -eq "None"
 $CopyDisabledArtifacts = Test-Path Env:COPY_DISABLED_ARTIFACTS
 
+if (-not $NothingToBuild) {
+    & $PSScriptRoot\Build.ps1
+}
+
 if ($NothingToBuild -or $CopyDisabledArtifacts) {
```

Comment:
Maybe this should be an elseif then?

Reply:
These ifs are not disjoint, so I don't think it'd be possible.

Reply:
How about

if(-not $NothingToBuild) {
..
} elseif ($CopyDisabledArtifacts) {
...
}

Reply:
We can't use `else` because we must to be able to enter both first and second if (as I said before, they're *not disjoint*)

Reply:
nvm I'm dumb